scripts/plot_dataframe.py

```diff
@@ -26,8 +26,13 @@
 from freqtrade import exchange
 import freqtrade.optimize as optimize
 
-logger = logging.getLogger(__name__)
+from typing import Dict, List, Any
+from freqtrade import persistence
+from sqlalchemy import create_engine
+from freqtrade.persistence import Trade
```

Comment:
i would keep all freqtrade-imports togther (basically moving typing and sqlalchemy together (and before the first freqtrade import ...)